tests/qauto/homework1.spec.js

```diff
@@ -0,0 +1,219 @@
+import { test, expect } from "@playwright/test";
+import { faker } from '@faker-js/faker';
+import RegistrationPage from "../../pageObjects/RegistrationPage";
+
+
+test.describe('Registration form test', () => {
+  let registrationPage;
+
+  test.beforeEach(async ({ page }) => {
+    await page.goto('/');
+    await page.locator('.btn-primary').click();
+    await expect(page.locator('.modal-content')).toBeVisible();
+    registrationPage = new RegistrationPage(page);
+  });
+
+  test('Positive', async ({ page }) => {
+    const signupName = faker.person.firstName();
+    const signupLastName = faker.person.lastName();
+    const signupEmail = faker.internet.email();
+    const signupPassword = faker.internet.password();
+
+    await registrationPage.fillRegistrationForm(signupName, signupLastName, signupEmail, signupPassword);
+
+    const name = await page.locator('#signupName').evaluate(element => element.value);
+    expect(name).toBe(signupName);
```

Comment:
use https://playwright.dev/docs/api/class-locatorassertions#locator-assertions-to-have-value